src/components/ImageView/index.js

```diff
@@ -117,67 +122,41 @@ class ImageView extends PureComponent {
             return;
         }
 
-        this.setState(prevState => ({
-            isZoomed: !prevState.isZoomed,
-            isMouseDown: false,
-        }));
+        // We hold for set isZoomed state if true
+        // Because when set isZoomed state can't getting actual position user clicked
```

Comment:
I don't follow your comments here. Can you rephrase? I think you are trying to say `onContainerPress` calculates where to scroll so we zoom in over here and not here, is that correct?
If so how does this rephrase sound?
```
// We won't set isZoomed true in the block below and instead set it in onContainerPress as that method calculates the scroll positions after zooming.
```